x/evm/keeper/grpc_query.go

```diff
@@ -470,6 +496,12 @@ func (k Keeper) TraceTx(c context.Context, req *types.QueryTraceTxRequest) (*typ
 	}, nil
 }
 
+func isUnsigned(ethTx *ethtypes.Transaction) bool {
+	r, v, s := ethTx.RawSignatureValues()
+
+	return (r == nil && v == nil && s == nil) || (r.Cmp(big.NewInt(0)) == 0 && v.Cmp(big.NewInt(0)) == 0 && s.Cmp(big.NewInt(0)) == 0)
```

Comment:
```suggestion
r.Cmp(big.NewInt(0)) == 0 
```
I think `r.Int64() == 0` is simpler